code/game/objects/items.dm

```diff
@@ -631,6 +631,42 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 			to_chat(M, "<span class='danger'>You go blind!</span>")
 	return TRUE
 
+///This proc runs when you hit someone with an item campable of performing a knee jerk reaction test, a measure of brain health
+/obj/item/proc/kneejerk(mob/living/carbon/M, mob/living/carbon/user)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Change to `carbon_target` or something similar